mock/auth_service.go

```diff
@@ -21,7 +21,7 @@ type AuthorizationService struct {
 	FindAuthorizationsFn       func(context.Context, platform.AuthorizationFilter, ...platform.FindOptions) ([]*platform.Authorization, int, error)
 	CreateAuthorizationFn      func(context.Context, *platform.Authorization) error
 	DeleteAuthorizationFn      func(context.Context, platform.ID) error
-	SetAuthorizationStatusFn   func(context.Context, platform.ID, platform.Status) error
+	UpdateAuthorizatioFn       func(context.Context, platform.ID, *platform.AuthorizationUpdate) error
```

Comment:
really nitpicky, but change this to `UpdateAuthorizationFn`

Reply:
thanks, fixed